### DISCUSSION TITLE
Help Wanted: How to load custom.ipxe in Docker from -v /config:/config ?

### DISCUSSION THREAD

**Discussion by jerrywoo96** [antonym/netboot.xyz#795] [Q&A]
I've managed to customised my custom menu and i want it to show in my main menu. How can i achieve this?

**Comment by Torpedro1978** [comment]
Hi!

I know what you mean, i'v been searching 5 hours this weekend! It awful when people think that it's good idea to skip "obvious" steps. This information ist missing here: https://netboot.xyz/docs/selfhosting/

This works for me now.

Configure your docker file to  include this volume:

`
    volumes:
      - /mysource/custom:/custom # set the custom menu source path 
`

In your boot.cfg set the variable custom_url  to the top under :global_vars

```
:global_vars
# set site name
set site_name my.site
# set custom_url 
set custom_url http://192.168.1.5:8030/custom/

```
